packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -682,7 +697,15 @@ class EditableText extends StatefulWidget {
        assert(rendererIgnoresPointer != null),
        assert(scrollPadding != null),
        assert(dragStartBehavior != null),
+       assert(
+          (contentInsertionMimeTypes != null && onContentInserted != null) ||
+          contentInsertionMimeTypes == null ||
+          onContentInserted == null
```

Comment:
Nit: Trailing comma here.

Reply:
Nit: Maybe add a message here explaining what the assert is checking, since the logic is hard to read for me at least.

Reply:
Also it seems to me like this assertion will always pass?  Maybe I'm reading it wrong.  Is this logic better?

```dart
(contentInsertionMimeTypes == null) == (onContentInserted == null)
```

Reply:
I fixed this assert, it is trying to make sure that if the mime types are provided, the `onContentInserted` should also be provided. Otherwise the OS will not show the toast for "<app> does not support inserting content here".